control-plane/roles/isolated-clusters/tasks/dns.yaml

```diff
@@ -0,0 +1,27 @@
+- name: Get kubeconfig
+  copy:
+    src: "files/kubeconfigs/kubeconfig.{{ cluster }}"
+    dest: "/tmp/kubeconfig.{{ cluster }}"
+
+- name: Create namespace {{ isolated_clusters_dns_namespace }}
+  k8s:
+    definition:
+      api_version: v1
+      kind: Namespace
+      metadata:
+        name: "{{ isolated_clusters_dns_namespace }}"
+        labels:
+          group: "{{ isolated_clusters_group_label }}"
+  environment:
+    K8S_AUTH_KUBECONFIG: "/tmp/kubeconfig.{{ cluster }}"
+
+- name: Deploy DNS service
+  k8s:
+    definition: "{{ lookup('template', 'dns/' + item) }}"
+    namespace: "{{ isolated_clusters_dns_namespace }}"
+  loop:
+    - coredns-configmap.yaml
+    - coredns-deployment.yaml
+    - coredns-service.yaml
+  environment:
+    K8S_AUTH_KUBECONFIG: "/tmp/kubeconfig.{{ cluster }}"
```

Comment:
```suggestion
- name: Deploy DNS service
  k8s:
    kubeconfig: "/tmp/kubeconfig.{{ cluster }}"
    template:
    - path: dns/coredns-configmap.yaml
    - path: dns/coredns-deployment.yaml
    - path: dns/coredns-service.yaml

Reply:
Revert beause:

```
  msg: 'Failed to create object: The `items` field in the body must be populated when calling methods on a ResourceList'
``` 

Maybe need to look at this one more time.